website/docs/r/ecs_cluster.html.markdown

```diff
@@ -23,6 +23,8 @@ resource "aws_ecs_cluster" "foo" {
 The following arguments are supported:
 
 * `name` - (Required) The name of the cluster (up to 255 letters, numbers, hyphens, and underscores)
+* `capacity_providers` - (Optional) The short name or full Amazon Resource Name (ARN) of one or more capacity providers to associate with the cluster.
```

Comment:
Might be good to denote at the beginning this is a list and that [valid values include `FARGATE` and `FARGATE_SPOT`](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_CreateCluster.html#ECS-CreateCluster-request-capacityProviders):

```suggestion
* `capacity_providers` - (Optional) List of short names or full Amazon Resource Names (ARNs) of one or more capacity providers to associate with the cluster. Valid values also include `FARGATE` and `FARGATE_SPOT`.
```